### ISSUE TITLE
Allow to use alias adresses (to deal with forwarded emails)

### ISSUE THREAD

**Issue by runmode** [dennisverspuij/tb-correctidentity#25]
Hi, your dropdown box only allows to select from accounts which had beed configured in TB. Please allow generic aliases to be added. Example:

Let's say I use 3 email addresses:
myname@mybusiness.com (default thunderbird mailbox account)
myname@mynewprivate.com (another thunderbird mailbox account, + a filter rule would copy it to myname@mybusiness.com)
myname@myoldprivate.com (forwarded to myname@myprivate.com)

Now, if an email to myname@myoldprivate.com (or myname@mynewprivate.com) arrives in the mailbox myname@mybusiness.com, I would like to configure that myname@mynewprivate.com is used for reply or forward.

**Comment by ldreier** [comment]
Hi @runmode ,

the dropdown list indeed lists the accounts/identities pairs configured for Thunderbird. So the addon just allows to select one automatically from the available list as you would manually do when composing the email.

Seems that you have not yet explored the additional identities feature of Thunderbird: [Using Identities](http://mzl.la/1BsOxfw)

So in your case you should configure [myname@mynewprivate.com](mailto:myname@mynewprivate.com) as an additional identity for the [myname@mybusiness.com](mailto:myname@mybusiness.com) account. Then it will appear as an option in the dropdown list.

**Comment by runmode** [comment]
Hi,
thank you for your hint to use TB identities. This solved my issue without the need of an add on. 
In fact I used identities before, but now I found that an identity needs to be set for each account that can receive emails for this particular identity. TB does not use the correct identity if it is set for another account besides of that beeing used when I reply to an email. Identities can and must be redundent for this reason. 
I hope my findings are correct and I do not miss anything else. Apologize my slow reaction.
